examples/resources/infra_settings/import.sh

```diff
@@ -0,0 +1 @@
+terraform import infra_settings.example example
```

Comment:
what does this do?

Reply:
This imports the resource into Terraform state. It's not strictly necessary for this resource since there's a single instance and the provider will automatically import it when configured.